src/server/lib/callback-handler.js

```diff
@@ -205,16 +223,20 @@ export default async function callbackHandler (sessionToken, profile, providerAc
         providerAccount.id,
         providerAccount.refreshToken,
         providerAccount.accessToken,
-        providerAccount.accessTokenExpires
+        providerAccount.accessTokenExpires,
+        ...providerAccount
```

Comment:
So actually I don't think you can spread an object like this onto a function's arguments.
It will throw something like:
> Uncaught TypeError: Found non-callable @@​iterator
    at <anonymous>:1:1

Maybe `...Object.values(providerAccount)` would work, but since it is a temporary solution (We **DEFINITELY** have to make this a named parameters function in 4.0... ), I think just pass it as `providerAccount`. If `providerAccount` has many items (I think it has like 10), it will be a nightmare to get anything out of it otherwise.

We will clean this up in an upcoming breaking release. What do you think?

Reply:
Hmm yeah you're right, I thought for sure you could do that. Maybe its only with an array then. 

Okay well lets scrap that part of this PR then, and maybe we can use this PR to do the aforementioned migration to named parameters for `linkAccount()`. What do you think?

Reply:
Yes, you can do it with an array, that is why I mentioned `Object.values`.

So in the end, we won't try to support this case in 3.x then? (Which I am fine with personally, but those who use the adapters could benefit anyway.)

Reply:
I don't think a simple file formatting is worthy enough change to make it to `main`, so I'll close this. Feel free to reopen, if you actually make the change discussed! :slightly_smiling_face:

(This is only to don't keep hanging PRs!)